src/main/java/seedu/modtracker/Task.java

```diff
@@ -43,22 +46,13 @@ public String getStatusIcon() {
         return (isDone ? "✓" : "✘");
```

Comment:
Not sure if really wanna use this as this is kind of buggy. Might need to update/ tweak cmd for it to show

Reply:
Is there a better alternative? I used the above method as using "\u2713" and "\u2718" violates the Gradle style check.

Reply:
Hmm, then for now we just use it. If it was encountered as a bug during testing then we change? Cos now this method is not tested yet right?

Reply:
Yeah, can.